src/NuGet.Core/NuGet.Packaging/docs/README.md

```diff
@@ -0,0 +1 @@
+NuGet's understanding of packages. Reading nuspec, nupkgs and package signing.
```

Comment:
````suggestion
# NuGet.Packaging

Install the `NuGet.Packaging` package to interact with `.nupkg` and `.nuspec` files, or, to verify packages' signatures. Refer to [`NuGet.Packaging`'s documentation](https://docs.microsoft.com/nuget/reference/nuget-client-sdk#nugetpackaging) for more information.

## Examples

Read a package:

```cs
using FileStream inputStream = new FileStream("MyPackage.nupkg", FileMode.Open);
using PackageArchiveReader reader = new PackageArchiveReader(inputStream);
NuspecReader nuspec = reader.NuspecReader;
Console.WriteLine($"ID: {nuspec.GetId()}");
Console.WriteLine($"Version: {nuspec.GetVersion()}");
Console.WriteLine($"Description: {nuspec.GetDescription()}");
Console.WriteLine($"Authors: {nuspec.GetAuthors()}");

Console.WriteLine("Dependencies:");
foreach (var dependencyGroup in nuspec.GetDependencyGroups())
{
    Console.WriteLine($" - {dependencyGroup.TargetFramework.GetShortFolderName()}");
    foreach (var dependency in dependencyGroup.Packages)
    {
        Console.WriteLine($"   > {dependency.Id} {dependency.VersionRange}");
    }
}

Console.WriteLine("Files:");
foreach (var file in reader.GetFiles())
{
    Console.WriteLine($" - {file}");
}
```
````